### PR TITLE
Revert "Revert "Remoevs the L6's burst fire since its effective long range dps is higher than the sniper""

### PR DESCRIPTION
**Pull request by SomeguyManperson**
Reverts yogstation13/Yogstation#12543

### CODE REVIEW

**Comment by maxion12345** [comment]
>theos makes a pr
>theos reverts his own pr
>theos reverts the revert to his own pr

what sort of sick fucking flex is this

**Comment by alexkar598** [comment]
> > theos makes a pr
> > theos reverts his own pr
> > theos reverts the revert to his own pr
> 
> what sort of sick fucking flex is this

Jamie merged theos' pr before it was ready, hence the do not merge label

**Comment by SomeguyManperson** [comment]
image

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**
